autoload/ledger.vim

```diff
@@ -267,6 +267,50 @@ endf "}}}
 
 " == helper functions ==
 
+" get a list of declared accounts in the buffer
+function! ledger#declared_accounts(...)
+  if a:0 == 2
+    let lnum = a:1
+    let end = a:2
```

Comment:
I would have thought that `end` is a reserved keyword in vim script. But this does not seem to be the case. 😉